cmd/snap/cmd_snap_op_test.go

```diff
@@ -39,6 +39,7 @@ func (s *SnapSuite) TestInstall(c *check.C) {
 			c.Check(r.URL.Path, check.Equals, "/v2/snaps/foo.bar")
 			c.Check(DecodedRequestBody(c, r), check.DeepEquals, map[string]interface{}{
 				"action":  "install",
+				"name":    "foo.bar",
```

Comment:
Just preserving the remaining of the tests untouched. We can fix it all in one swap next week.